modules/services/wlsunset.nix

```diff
@@ -20,15 +22,19 @@ in {
     };
 
     latitude = mkOption {
-      type = types.str;
+      type = with types; nullOr (either float int);
```

Comment:
Just noticed: changing this type of the pre-existing options is backwards-incompatible, i.e. this would break existing configurations. As much as float and int make more sense than str, I think str should continue to be allowed for these.

Reply:
I didn't consider backward compatibility originally but I can make that change